### PR TITLE
Import from OBS

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

File: rpm/grubby.spec [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+%make_install
+
+# >> install post
+# << install post
+
+
+%files
+%defattr(-,root,root,-)
+# >> files
+%doc COPYING
```
Comment (Thaodan) [inline rpm/grubby.spec]: This should be %license.

File: rpm/grubby.spec [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+/sbin/installkernel
```
Comment (Thaodan) [inline rpm/grubby.spec]: Use %{_sbindir}.

File: rpm/grubby.spec [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+# << build pre
+
+
+make %{?jobs:-j%jobs}
+
+# >> build post
+# << build post
+
+%install
+rm -rf %{buildroot}
```
Comment (Thaodan) [inline rpm/grubby.spec:67]: This can be skipped %clean will cleanup %{buildroot}.

File: rpm/grubby.spec [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+%patch5 -p1
+# >> setup
+# << setup
+
+%build
+# >> build pre
+# << build pre
+
+
+make %{?jobs:-j%jobs}
```
Comment (Thaodan) [inline rpm/grubby.spec:61]: Use %make_build.

File: rpm/grubby.spec [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+grubby  is  a command line tool for updating and displaying information about 
+the configuration files for the grub, lilo, elilo (ia64),  yaboot (powerpc)  
+and zipl (s390) boot loaders. It is primarily designed to be used from scripts
+which install new kernels and need to find information about the current boot 
+environment.
+
+
+
+%prep
+%setup -q -n %{name}-%{version}/%{name}
```
Comment (Thaodan) [inline rpm/grubby.spec:39]: We can use %autosetup here.

**Review by Thaodan — CHANGES_REQUESTED** [review]

**Comment by martyone** [comment]
@Thaodan it's good respect the scope of a task during review and not bloat the review with "fix this and that while here". Fixed just the hand-written part of the SPEC.

File: rpm/grubby.spec (hunk 3 above)
Comment (martyone) [reply rpm/grubby.spec:67]: Autogenerated, out of scope.

File: rpm/grubby.spec (hunk 4 above)
Comment (martyone) [reply rpm/grubby.spec:61]: Autogenerated, out of scope.

File: rpm/grubby.spec (hunk 5 above)
Comment (martyone) [reply rpm/grubby.spec:39]: Autogenerated, out of scope.

**Review by Thaodan — APPROVED** [review]

**Review by rainemak — APPROVED** [review]

**Pull request merged**